content/platform/overview/registering-an-event-against-a-document-profile-asset/index.md

```diff
@@ -0,0 +1,339 @@
+    event_attributes:
+      arc_description: Withdraw My First Document
+      arc_display_type: Withdraw
+    confirm: true
+```
+{{< /tab >}}
+{{< tab name="JSON" >}}
+
+Use the attibute/value pairs in the example below to register a `Withdraw` event.
+```json
+{
+  "behaviour": "RecordEvidence",
+    "operation": "Record",
+    "asset_attributes": {
+        "document_status":"Withdrawn"
+    },
+    "event_attributes": {
+        "arc_description":"Withdraw My First Document",
+        "arc_display_type":"Withdraw"
+    }
+}
+```
+{{< /tab >}}}
+{{< /tabs >}}
+<br>
+4. Record Event.
+
+An Event type for generic events that are not part of the Document Profile lifecycle. The asset and event attributes are in separate tabs in this Event type.
+
+See [Creating an Event Against an Asset](/platform/overview/creating-an-event-against-an-asset/) for more information on this event type.
+{{< tabs name="record_event" >}}
+{{{< tab name="UI" >}}
+
+{{< img src="RecordEvent.png" alt="Rectangle" caption="<em>Submitting the Event</em>" class="border-0" >}}
+
+{{< /tab >}}
+{{< tab name="YAML" >}}
+Use the [archivist_runner](https://python.rkvst.com/runner/index.html) to run your YAML file!
+ 
+```yaml
+---
+steps:
+  - step:
+      action: EVENTS_CREATE
+      description: Record event against My First Document.
+      asset_label: assets/<asset-id>
+    operation: Record
+    behaviour: RecordEvidence
+    event_attributes:
+      arc_description: Document review
+      arc_display_type:  Review
+    confirm: true
+```
+{{< /tab >}}
+{{< tab name="JSON" >}}
+```json
+{
+  "operation": "Record",
+  "behaviour": "RecordEvidence",
+  "event_attributes": {
+    "arc_description": "Document Review",
+    "arc_display_type": "Review",
+  },
+}
+
+```
+{{< /tab >}}}
+{{< /tabs >}}
+<br>
+5. Register the Event
+{{< tabs name="register_event" >}}
+{{{< tab name="UI" >}}
+Click on the `Record Event` button to register the event.
+{{< img src="RecordEvent.png" alt="Rectangle" caption="<em>Submitting the Event</em>" class="border-0" >}}
+{{< /tab >}}
+{{< tab name="YAML" >}}
+Use the [archivist_runner](https://python.rkvst.com/runner/index.html) to run your YAML file!
+ 
+```bash
+$ archivist_runner \
+      -u https://app.rkvst.io \
+      --client-id <your-client-id> \
+      --client-secret <your-client-secret> \
+      my_first_document_event.yaml
+```
+{{< /tab >}}
+{{< tab name="JSON" >}}
+Use the curl command to run your JSON file! See instructions for [creating your `BEARER_TOKEN_FILE`](/developers/developer-patterns/getting-access-tokens-using-app-registrations/) here.
+ 
+```bash
+curl -v -X POST \
+    -H "@$BEARER_TOKEN_FILE" \
+    -H "Content-type: application/json" \
+    -d "@/path/to/jsonfile" \
+    https://app.rkvst.io/archivist/v2/assets/<asset-id>/events
+```
+{{< /tab >}}}
+{{< /tabs >}}
+<br>
+6. View your Event details. 
+
+{{< tabs name="view_event" >}}
+{{{< tab name="UI" >}}
+Click the Event row to inspect the Event:
```

Comment:
THis is very clear about where the user has moved to, or what to click. I'm wary of too many screengrabs, but this seems to miss a step or isn't clear about where in the process we are.